src/bridgestuff/BridgeSettingsDiscord.ts

```diff
@@ -4,6 +4,7 @@ export interface BridgeSettingsDiscordProperties {
 	relayJoinMessages: boolean;
 	relayLeaveMessages: boolean;
 	crossDeleteOnTelegram: boolean;
+	disableWebPreviewOnTelegram?: boolean;
```

Comment:
that doesn't seem to work

Reply:
![Screenshot_20230911_194311](https://github.com/TediCross/TediCross/assets/42044185/6f012878-2e71-4a5a-a9c7-08b4233f0ca2)

It works.
1st Message sent from DS when option is true.
2nd - when it's false.
It does not depend on Discord's hiding method using <> quotes.

Reply:
ok weird, maybe I did something wrong, but the `>;` bug is still there.

Reply:
It's because of this replacements:
`export const escapeHTMLSpecialChars = R.compose(
	R.replace(/>/g, "&gt;"),
	R.replace(/</g, "&lt;"),
	R.replace(/&/g, "&amp;")
);`

Reply:
Uh.. OK I'm gonna fix that and then we're gonna merge and release.

Reply:
I can't find proper quick-fix for that.

But I'd suggest one small change - remove ONE symbol "`;`"
`R.replace(/>/g, "&gt")`
This works pretty well.

To get fully correct HTML we can later replace all "`$gt`" with "`$gt;`" right before returning the result here
`return htmlCleanup(html, settings);`

Reply:
hmm that gets even more weird. i rather set that on a later todo